hadoop-connector-examples/README.md

```diff
@@ -38,15 +38,40 @@ Hadoop (verified with Hadoop 2.8.3 on Ubuntu 16.04)
    export PRAVEGA_STREAM=<stream_name> # e.g. myStream
    export CMD=wordcount # so far, can also try wordmean and wordmedian
 
-3. make sure below dirs are empty
+3. make sure input dir is empty
    hadoop fs -rmr ${HADOOP_EXAMPLES_INPUT_DUMMY}
-   hadoop fs -rmr ${HADOOP_EXAMPLES_OUTPUT}
 
-4. generate words into pravega
-   hadoop jar ${HADOOP_EXAMPLES_JAR} randomtextwriter -D mapreduce.randomtextwriter.totalbytes=32000 ${HADOOP_EXAMPLES_INPUT_DUMMY} ${PRAVEGA_URI} ${PRAVEGA_SCOPE} ${PRAVEGA_STREAM}
+4. run hadoop commands
 
-5. run hadoop command
-   hadoop jar ${HADOOP_EXAMPLES_JAR} ${CMD} ${HADOOP_EXAMPLES_INPUT_DUMMY} ${PRAVEGA_URI} ${PRAVEGA_SCOPE} ${PRAVEGA_STREAM} ${HADOOP_EXAMPLES_OUTPUT}
+   4.1 to process all events in the stream
```

Comment:
I tested the example and it works. Maybe the way of instantiating stream cuts is not the most intuitive one from a user's perspective. I would suggest to clarify a little bit more in the `README` file what is the user achieving executing these steps and at least add a reference to the concept of stream cut (see http://pravega.io/docs/latest/streamcuts/).

Reply:
accepted. added this link